content/en/templates/taxonomy-templates.md

```diff
@@ -126,9 +126,13 @@ Taxonomies can be ordered by either alphabetical key or by the number of content
 
 ```
 <ul>
-  {{ $data := .Data }}
+  {{ $type := .Type }}
   {{ range $key, $value := .Data.Terms.Alphabetical }}
-  <li><a href="{{ $.Site.LanguagePrefix }}/{{ $data.Plural }}/{{ $value.Name | urlize }}"> {{ $value.Name }} </a> {{ $value.Count }} </li>
+  {{ $name := .Name }}
+  {{ $count := .Count }}
+  {{ with $.Site.GetPage "taxonomy" (printf "%s/%s" $type $name) }}
```

Comment:
Thanks for your work. This will actually need one more iteration. While above works on Hugo 0.45, it do so because of forced backward compatibility (as I later learned).

The *correct* way of doing this in Hugo v0.45 onwards is to have **just one argument** to `.Site.GetPage`.. as follows:

```html
{{ with $.Site.GetPage (printf "/%s/%s" $type $name) }}
```

Reply:
Fixed.